app/models/piece.rb

```diff
@@ -14,7 +14,7 @@ def move_to!(params)
       # Step 2: Check to see if the piece is of the oppposite color.
       if color != enemy_piece[:color]
         # Step 3: Remove piece from the board.
-        enemy_piece.update_attributes(horizontal_position: nil, vertical_position: nil)
+        enemy_piece.destroy
```

Comment:
@trustinruss, aside from the game_controller spec, the tests were running fine for me as well. And nope, no indentation on config/routes.rb. When I'm on line 6, a left error takes me up to line 5 and a right arrow takes me down to line 7. Deleting the space altogether makes the error go away as does commenting it out, but I'm just going to leave it as is and ignore Rubocop on that one...